js/pages/characterizations/components/feature-analyses/feature-analysis-view-edit.js

```diff
@@ -12,6 +12,7 @@ define([
     'text!./feature-analysis-view-edit.html',
     'appConfig',
     'atlas-state',
+    'services/EventEmitter',
```

Comment:
Note on the 'EventEmitter':  in the framework we're using, we have capabilities for pub-sub notificaiton via observable, or computedObservable, which you can subscribe and valueHasMutated to signal.  I'd prefer if we didn't add a new dependency to the application that effectively does the same thing, but in a different way.